lti_consumer/tests/unit/test_utils.py

```diff
@@ -11,7 +11,7 @@
 FAKE_USER_ID = 'fake_user_id'
 
 
-def make_xblock(xblock_name, xblock_cls, attributes):
+def make_xblock(xblock_name, xblock_cls, attributes, is_past_due_patch=None):
```

Comment:
@shimulch You should not patch this method. When doing testing, make sure to set the due date properly and then patch the current time.

Reply:
@giovannicimolin Removed the function patch. Instead patching ``django.utils.timezone`` while testing.